spec/rubocop/path_util_spec.rb

```diff
@@ -10,6 +10,11 @@
     it 'supports custom base paths' do
       expect(described_class.relative_path('/foo/bar', '/foo')).to eq('bar')
     end
+
+    it 'works for different drives' do
+      expect(described_class.relative_path('D:/foo/bar', 'C:/foo'))
+        .to eq('D:/foo/bar')
+    end
```

Comment:
I guess the test should run on Windows only.
`relative_path` methods uses `File.expand_path`, and `FIle.expand_path('D:/foo/bar')` returns unexpected value in Linux.

```
$ pwd 
/tmp/tmp.tVst8PbbvU

$ ruby -e 'p File.expand_path("D:/foo/bar")'
"/tmp/tmp.tVst8PbbvU/D:/foo/bar"
```

So the test case makes no sense in Linux.

Reply:
It doesn't hurt. So on linux the test case wouldn't have failed even without the fix.

Do you still prefer to limit the test to Windows only?

Reply:
> Do you still prefer to limit the test to Windows only?

Yes, because it doesn't check anything in Linux. So I think it is unnecessary in Linux.

And can you add a test case that checks to work for same drives? For example: `relative_path('D:/foo/bar', 'D:/foo')`